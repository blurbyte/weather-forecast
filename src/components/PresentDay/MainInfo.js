import React from 'react';
import styled from 'styled-components';

import BaseRow from '../Row';
import WeatherIcon from './WeatherIcon';
import Temperature from './Temperature';

const Row = styled(BaseRow)`
  margin-top: 0.4rem;
`;

const MainInfo = () => (
  <Row>
    <WeatherIcon />
    <Temperature>19&deg;</Temperature>
  </Row>
);

export default MainInfo;
