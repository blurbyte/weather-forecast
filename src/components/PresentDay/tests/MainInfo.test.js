import React from 'react';
import renderer from 'react-test-renderer';

import MainInfo from '../MainInfo';

test('renders correctly', () => {
  const props = {
    temp: 12.34,
    weatherId: 800,
    isDaytime: true
  };

  const component = renderer.create(<MainInfo {...props} />);
  expect(component.toJSON()).toMatchSnapshot();
});
