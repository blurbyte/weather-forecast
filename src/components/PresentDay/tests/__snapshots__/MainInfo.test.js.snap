// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="sc-bxivhb ehicqO sc-bdVaJa iInree"
>
  <div
    className="sc-bwzfXH iUDDfa"
  >
    <svg
      viewBox="0 0 110 110"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M80 56.54c0 13.807-11.193 25-25 25s-25-11.193-25-25 11.193-25 25-25 25 11.193 25 25zm-25-40a5 5 0 1 0 .001 10.001A5 5 0 0 0 55 16.54zm-25.39 9.241a5 5 0 1 0 6.429 7.66 5 5 0 0 0-6.429-7.66zm-13.509 23.4a5 5 0 0 0 4.056 5.792 5 5 0 1 0 1.736-9.848 5 5 0 0 0-5.792 4.056zm4.692 26.608a5 5 0 1 0 8.66-5 5 5 0 0 0-8.66 5zM41.49 93.157a5 5 0 1 0 3.42-9.398 5 5 0 0 0-3.42 9.398zm27.02 0a5 5 0 1 0-3.421-9.397 5 5 0 1 0 3.421 9.397zm20.697-17.368a5 5 0 1 0-8.66-5 5 5 0 0 0 8.66 5zm4.692-26.608a5.001 5.001 0 0 0-9.848 1.736 5.001 5.001 0 0 0 9.848-1.736zm-13.509-23.4a5 5 0 1 0-6.428 7.66 5 5 0 0 0 6.428-7.66z"
        fill="#fae36b"
      />
    </svg>
  </div>
  <p
    className="sc-htpNat dGZyVn"
  >
    19Â°
  </p>
</div>
`;
